### ISSUE TITLE
signalOnStop broken (tested on web UI)

### ISSUE THREAD

**Issue by fishnux** [yohamta/dagu#328]
Hi,

signalOnStop is being ignored when stopping through the Web UI. Haven't tested other methods to stop the DAG

```
steps:
  - name: step1
    command: bash
    script: |
      for s in {1..64}; do trap "echo trap $s" $s; done
      sleep 60
    signalOnStop: "SIGINT"
```
Stop the DAG shortly after starting
Log:
```
Terminated
trap 15
trap 17
```
15 is SIGTERM
17 is SIGCHLD

**Comment by yohamta** [comment]
Thank you for reporting the issue! I will take a look.

**Comment by yohamta** [comment]
@fishnux This bug is fixed in v1.8.6. Apologies for the inconvenience.

**Comment by fishnux** [comment]
Thank you so much for the fast resolution @yohamta! No need to apologise, it's awesome you keep working on it 👍
